File: Bin/PLSamples/Data/Scripts/Calculate.js
//[-------------------------------------------------------]
//[ Global variables                                      ]
//[-------------------------------------------------------]
var g_Factor = 1;


//[-------------------------------------------------------]
//[ Global functions                                      ]
//[-------------------------------------------------------]
function scriptFunction(a)
{
	return a;
}

function callCpp(a)
{
	return cppFunction(a) + cppMethod(a) + cppScriptFunction(a);
}

function getFactor()
{
	return g_Factor;
}

function setFactor(a)
{
	g_Factor = a;
}

function calculate(a, b)
{
	return a * b * g_Factor;
}

File: Bin/PLSamples/Data/Scripts/Calculate.lua
--[-------------------------------------------------------]
--[ Global variables                                      ]
--[-------------------------------------------------------]
g_Factor = 1


--[-------------------------------------------------------]
--[ Global functions                                      ]
--[-------------------------------------------------------]
function scriptFunction(a)
	return a
end

function callCpp(a)
	return FirstNamespace.SecondNamespace.cppFunction(a) + cppMethod(a) + cppScriptFunction(a)
end

function getFactor()
	return g_Factor
end

function setFactor(a)
	g_Factor = a
end

function calculate(a, b)
	return a * b * g_Factor
end

function returnMyString(s)
	return cppStringFunction(s)
end
